site/src/css/custom.css

```diff
@@ -8,6 +8,7 @@
   --ifm-color-primary: var(--uitk-text-primary-foreground);
   --ifm-font-family-base: var(--uitk-typography-fontFamily);
   --card-background: #10212b;
+  --image-border: #c5c9d0;
```

Comment:
This is more of a design question but is there any way we could use a design token here instead of a custom value?

Reply:
seems this is container-border-primary in figma